### DISCUSSION TITLE
Bug - Narration, function naming conventions

### DISCUSSION THREAD

**Discussion by jqmg** [GitbookIO/community#380] [0. General bugs and questions]
Using the work "underscore" when describing a function name actually includes "underscore" in the documented function name. Given that using underscores is a somewhat common practice in function naming, Gitbook should know to exclude that wording. Example, I was working with a `mg_field` function in VS Code and Gitbook's capture picked this up as `mg_underscore_field`.

Excluding the word "underscore" in narration the function naming convention defaults to camelcase.

**Comment by rodrcastro** [comment]
Hey @jqmg 👋 

Thank you for your collaboration and feedback. I've sent your comments to our team internally so we can work on this during this beta process.

It should be something we take into consideration when doing a capture.
